pkg/config/legacy/converter_test.go

```diff
@@ -205,6 +205,29 @@ func TestDefaultValues(t *testing.T) {
 	assert.Equal(t, true, config.Datadog.GetBool("hostname_fqdn"))
 }
 
+func TestTraceIgnoreResources(t *testing.T) {
+	require := require.New(t)
+
+	cases := []struct {
+		config   string
+		expected []string
+	}{
+		{"r1", []string{"r1"}},
+		{`"r1","r2,"`, []string{"r1", "r2,"}},
+		{`"r1"`, []string{"r1"}},
+		{"r1,r2", []string{"r1", "r2"}},
```

Comment:
You should probably use them for every config to stay consistent and make reading easier